app/controllers/api/v1/merchants_controller.rb

```diff
@@ -1,10 +1,10 @@
 class Api::V1::MerchantsController < ApplicationController
 
   def index
-    render json: Merchant.all
+    @merchants = Merchant.all
   end
 
   def show
-    render json: Merchant.find(params[:id])
+    @merchant = Merchant.find(params[:id])
```

Comment:
Hate to lose all the work we did getting `respond_with` to work, but this is efficient.